### DISCUSSION TITLE
What is the best practice for creating casks when there is an Intel AND Apple Silicon version

### DISCUSSION THREAD

**Discussion by slifty** [Homebrew/discussions#2682] [Writing Formulae/Casks]
I'm about to open a cask request for [OpenRGB](https://openrgb.org/releases.html) which offers native builds for both Intel and Apple Silicon.

I didn't see guidance in the cask-creation README about this -- should I create two separate casks, one for each?  What is the recommended naming convention to differentiate which is which?

**Comment by miccal** [comment]
See the Cask file for [`slack`](https://github.com/homebrew/homebrew-cask/blob/master/Casks/slack.rb) as an example.
